crates/autopilot/src/domain/auction/order.rs

```diff
@@ -10,8 +10,9 @@ pub struct Order {
     pub buy_token: H160,
     pub sell_amount: U256,
     pub buy_amount: U256,
-    pub solver_fee: U256,
     pub user_fee: U256,
+    // Same as user_fee, but without subsidies. This value should be used to score solutions.
+    pub scoring_fee: U256,
```

Comment:
Should we actually completely remove `scoring_fee`?  Right now we don't use subsidies so scoring_fee=user_fee and we are also moving to rank by surplus which will ignore fee when scoring?

Reply:
Decided to remove scoring fee.